docs/manual/scala/guide/cluster/PersistentEntityCassandra.md

```diff
@@ -68,8 +68,6 @@ lagom.persistence.read-side.cassandra {
 
 With these properties set to `false`, if the keyspaces or tables are missing at startup your service will log an error and fail to start.
 
-It is not possible to disable automatic creation of the offset store table at this time.
-
```

Comment:
It is quite possible the exact sentence exists in `docs/manual/java/guide/...`.

We keep duplicate pages of the docs so it's easier to diverge them and explain each API separately. The tradeoff is that we need to maintain two copies of the duplicate sections.